left_rgbd_position: [-0.1275, 0.23, 0.0]
left_rgbd_rotation: [0.0, 0.0, 0.0]

center_rgbd_position: [0.0, 0.0, 0.0]
center_rgbd_rotation: [0.0, 0.0, 0.0]

right_rgbd_position: [-0.1275, -0.23, 0.0]
right_rgbd_rotation: [0.0, 0.0, 0.0]